header.go

```diff
@@ -43,6 +43,61 @@ type Header struct {
 	rawTLVs            []byte
 }
 
+// HeaderProxyFromAddrs creates a new PROXY header from a source and a
+// destination address. The latest protocol version is used.
+func HeaderProxyFromAddrs(sourceAddr, destAddr net.Addr) *Header {
+	h := &Header{
+		Version: 2,
```

Comment:
This should be an optional argument that if unset then default to 2.

Reply:
Makes sense, will change this.